packages/remix/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@frames.js/remix",
```

Comment:
wondering if this shouldn't just be in the main package similar to how we did next - what are your thoughts?

Reply:
I think the problem would be to have mixed `peerDependencies` where you would have next, remix, express...